pymeasure/instruments/signalrecovery/dsp7265.py

```diff
@@ -22,228 +22,115 @@
-        "IE", "IE %d",
-        """Controls the oscillator reference. Can be "internal",
-        "external rear" or "external front" """,
-        validator=strict_discrete_set,
-        values=REFERENCES,
-        map_values=True
-    )
-
-    @property
-    def sensitivity(self):
-        """ A floating point property that controls the sensitivity
-        range in Volts (for voltage mode) or Amps (for current modes).
-        When in Volts it takes discrete values from 2 nV to 1 V. When in
-        Amps it takes discrete values from 2 fA to 1 µA (for normal current
-        mode) or up to 10 nA (for low noise current mode).
-        This property can be set. """
-        return self.values("SEN.")[0]
-
-    @sensitivity.setter
-    def sensitivity(self, value):
-        # get the voltage/current mode:
-        imode = self.IMODES.index(self.imode)
-
-        # Scale the sensitivities to the correct range for voltage/current mode
-        sensitivities = [s * self.SEN_MULTIPLIER[imode] for s in self.SENSITIVITIES]
-        if imode == 2:
-            sensitivities[0:7] = [np.nan] * 7
-
-        # Check and map the value
-        value = truncated_discrete_set(value, sensitivities)
-        value = sensitivities.index(value)
+        """Control the voltage of the DAC4 output on the rear panel.
 
-        # Set sensitivity
-        self.write("SEN %d" % value)
-
-    imode = Instrument.control(
-        "IMODE", "IMODE %d",
-        """ Property that controls the voltage/current mode. can be
-        'voltage mode', 'current mode', or 'low noise current mode' """,
-        validator=strict_discrete_set,
-        values=IMODES,
-        map_values=True
-    )
-    slope = Instrument.control(
-        "SLOPE", "SLOPE %d",
-        """ A integer property that controls the filter slope in
-        dB/octave, which can take the values 6, 12, 18, or 24 dB/octave.
-        This property can be set. """,
-        validator=truncated_discrete_set,
-        values=[6, 12, 18, 24],
-        map_values=True
-    )
-    time_constant = Instrument.control(
-        "TC", "TC %d",
-        """ A floating point property that controls the time constant
-        in seconds, which takes values from 10 microseconds to 50,000
-        seconds. This property can be set. """,
-        validator=truncated_discrete_set,
-        values=TIME_CONSTANTS,
-        map_values=True
+        Valid values are floating point numbers between -12 to 12 V.
+        """,
+        validator=strict_range,
+        values=[-12, 12]
     )
 
-    def set_voltage_mode(self):
-        self.write("IMODE 0")
-
-    def setDifferentialMode(self, lineFiltering=True):
-        """Sets lockin to differential mode, measuring A-B"""
-        self.write("VMODE 3")
-        self.write("LF %d 0" % 3 if lineFiltering else 0)
-
-    def setChannelAMode(self):
-        self.write("VMODE 1")
-
     @property
     def adc3(self):
+        """Reads the ADC3 input voltage."""
```

Comment:
Docstrings should be in imperative mode.
This is a property and should follow property docstring style (even though it is not created by our property creators `control`):
```suggestion
        """Measure the ADC3 input voltage."""
```

Reply:
Updated in last commit